spec/features/user_can_view_other_user_profiles_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.feature "Edit Posts", type: :feature do
+
+  before(:each) do
+    successful_sign_up
+    click_button "Log Out"
+    first_user_post
+    click_button "Log Out"
+    successful_sign_up_second_user
+    click_button "Log Out"
+    second_user_post
+  end
```

Comment:
the before steps don't look right, I think we only need successful_sign_up or successful_sign_in ?

Reply:
Thanks!